rfcs/2020-08-26-3191-host-metrics.md

```diff
@@ -0,0 +1,204 @@
+# RFC 3191 - 2020-08-26 - Collecting host-based metrics
+
+This RFC is to introduce a new metrics source to consume host-based metrics. The high level plan is to implement one (or more?) sources that collect CPU, disk, network, and memory metrics from Linux-based hosts.
+
+## Scope
+
+This RFC will cover:
+
+- A new source for host-based metrics, specifically:
+  - CPU
+  - Memory
+  - Disk
+  - Network
+- Collection on Linux, Windows, OSX.
+
+This RFC will not cover:
+
+- Other host metrics.
+- Other platforms
+
+## Motivation
+
+Users want to collect, transform, and forward metrics to better observe how their hosts are performing.
+
+## Internal Proposal
+
+Build a single source called `host_metrics` (name to be confirmed) to collect host/system level metrics.
```

Comment:
I'm 👍 on `host_metrics` unless others disagree. An alternative could be `node_metrics`, but that seems less precise to me. I'm curious why Prometheus adopted this nomenclature.